contracts/Cargo.toml

```diff
@@ -26,6 +26,9 @@ basic-analyzer = { path = "contracts/basic-analyzer", version = "0.1.0", feature
 ethereum-analyzer = { path = "contracts/ethereum-analyzer", version = "0.1.0", features = [
   "library",
 ] }
+contracts-core = { path = "contracts/contracts-core", version = "0.1.0", features = [
```

Comment:
maybe we can call it "analyzers-core" since it's not for all kind of contracts

Reply:
@Pitasi, are you thinking about moving it out of the contracts workspace? Or do you want to move it outside of the contracts folder within the workspace?

Reply:
I renamed to `analyzers-core` though

Reply:
> Or do you want to move it outside of the contracts folder within the workspace?

i think this. They can all be in the same workspace, so the contracts inside a "contracts" subfolder and this analyzers-core inside a "lib" or "shared" or "common" subfolder

Reply:
Done ✅